src/main/java/be/seeseemelk/mockbukkit/inventory/InventoryMock.java

```diff
@@ -97,8 +88,7 @@ public void assertTrueForSome(@NotNull Predicate<ItemStack> condition)
 	 *
 	 * @param item The itemstack to compare everything to.
 	 */
-	public void assertContainsAny(@NotNull ItemStack item)
-	{
+	public void assertContainsAny(@NotNull ItemStack item) {
```

Comment:
You broke the formatting here.

```suggestion
	public void assertContainsAny(@NotNull ItemStack item)
	{
```